zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -1,7 +1,8 @@
 """Xiaomi aqara button sensor."""
 
 from zigpy.profiles import zha
-from zigpy.zcl.clusters.general import Basic, Identify, MultistateInput, OnOff
+import zigpy.types as types
```

Comment:
Hi, thanks for the work, looking forward to getting support for the `b1acn02`.

Would changing lines 3-4 to 
```py
from zigpy import types
from zigpy.profiles import zha
```
fix the linting issue in [this check](https://github.com/zigpy/zha-device-handlers/actions/runs/8949592850/job/24584064765?pr=2871#step:8:24)?

Reply:
done . thnx wats next to do?

Reply:
Thanks!